dev/ci/ci-coq-library-undecidability.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download coq-library-undecidability
+
+if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
+
+install_ssreflect
```

Comment:
This doesn't exist anymore, you need to depend on mathcomp instead

Reply:
No. I followed the manual, which recommends looking at the Quickchick PR. I'll remove it

Reply:
Maybe worth updating, I noticed several things in this PR which don't look like things look now in the files: https://github.com/coq/coq/blob/master/dev/ci/README-users.md?plain=1#L78